node.gyp

```diff
@@ -322,6 +322,7 @@
           'dependencies': [
             'deps/v8_inspector/third_party/v8_inspector/platform/'
                 'v8_inspector/v8_inspector.gyp:v8_inspector_stl',
+            'v8_inspector_compress_protocol_json#host',
```

Comment:
nit: is there a reason for the inconsistent indent between this and the two lines above?